recipe/build.sh

```diff
@@ -25,10 +25,14 @@ cmake ${CMAKE_ARGS} -GNinja \
     -DIGRAPH_ENABLE_LTO=$ENABLE_LTO \
     -DIGRAPH_ENABLE_TLS=1 \
     -DBUILD_SHARED_LIBS=on \
+    -DBLAS_LIBRARIES="$PREFIX/lib/libblas${SHLIB_EXT}" \
+    -DLAPACK_LIBRARIES="$PREFIX/lib/liblapack${SHLIB_EXT}" \
     ..
 
-cmake --build . --config Release --target igraph
-cmake --build . --config Release --target check
+cmake --build . --config Release --target igraph -- -j${CPU_COUNT}
```

Comment:
Why did you explicitly add `-j${CPU_COUNT}` here by the way? It was my understanding that Ninja detects the number of cores itself and parallelizes the build automatically.

Reply:
It does. `conda-build` does the same and sets `CPU_COUNT` as well, but the user can override this whereas there's no way to override ninja's builtin count except for setting `NINJAFLAGS` which is not passed through by conda-build.

Reply:
OK, so might it then be better to pass this option in all `cmake` (and `ctest`) invocations?

Reply:
Yes